tests/test_objective_funs.py

```diff
@@ -535,6 +537,74 @@ def test_quadratic_flux(self):
         # for this to pass with rtol=1e-3, the source resolution needs to be quite high
         np.testing.assert_allclose(f, Bnorm, rtol=1e-2, atol=1e-2)
 
+    def test_quadratic_flux_optimization(self):
+        """Test optimization using quadratic flux."""
+        eq = Equilibrium(M=2, L=2)
```

Comment:
would be good to comment this a bit to say what the test is doing (optimizing to reduce the Phi_mn which produce non-axisymmetric fields)